foundation/keystore/keystore.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // KeyStore represents an in memory store implementation of the
-// KeyStorer interface for use with the auth package.
+// KeyStore interface for use with the auth package.
```

Comment:
This is wrong. It should stay KeyStorer. I am sure the interface is named that.

Reply:
I based this on the module level comment which mentions `auth.KeyStore` interface. Honestly I was not able to find a reference for `Keystorer` interface.

Reply:
The interface is called KeyStore. This comment should remain as is.

Reply:
oh, i see you fixed it actually.

Reply:
This should be called KeyStorer so I will fix that after this PR.